Ex1/week5_1.py

```diff
@@ -0,0 +1,51 @@
+import datetime
+import os
+import random
+
+# That's the way
+
+
+def find_files_with_deep_prefix(directory_path):
+    """
+    This function gets the path to a directory and return a list of files that start with the string "deep"
+    :return: List of files that start with "deep"
+    """
+    return [file for file in os.listdir(directory_path) if file.startswith("deep")]
+
+
+# I don't have vinaigrette
+
+
+def get_random_date_between(first_date, second_date):
```

Comment:
Add type hints, document parameters everywhere you code should look like this - 
![Capture](https://user-images.githubusercontent.com/13467749/160242248-5b851ac4-fddb-4cb7-937b-6fc7fd7df187.JPG)